crates/notary/server/tee/run-gramine-local.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# This script is used to build and run the Gramine manifest for the Notary server in a local development environment.
```

Comment:
like above, i feel that these explanations should be in the tee readme so that people can get all the infos in one place on how to run these files